components/layout_2020/query.rs

```diff
@@ -263,9 +263,9 @@ pub fn process_resolved_style_request_for_unstyled_node<'dom>(
     );
     let style = styles.primary();
     let longhand_id = match *property {
-        PropertyId::LonghandAlias(id, _) | PropertyId::Longhand(id) => id,
-        PropertyId::ShorthandAlias(id, _) | PropertyId::Shorthand(id) => {
-            return shorthand_to_css_string(id, style);
+        PropertyId::NonCustom(id) => match id.longhand_or_shorthand() {
+            Ok(lh) => lh,
+            Err(sh) => return shorthand_to_css_string(sh, style),
```

Comment:
I would go with `longhand` and `shorthand` here instead of `lh` and `sh`.

Reply:
Changed to `longhand_id` and `shorthand_id`.